utils/config/daos_server.yml

```diff
@@ -355,48 +358,51 @@
 #  env_vars:
 #      - CRT_TIMEOUT=100
 #
-#  # Define a pre-configured mountpoint for storage class memory to be used
-#  # by this engine.
-#  # Path should be unique to engine instance (can use different subdirs).
-#
-#  scm_mount: /mnt/daos/2
-#
-#  # Backend SCM device type. Either use PMem (Intel(R) Optane(TM) persistent
-#  # memory) modules configured in interleaved mode or emulate SCM with a
-#  # tmpfs running in RAM.
-#  # Options are:
-#  # - "dcpm" for real SCM (preferred option), scm_size is ignored
-#  # - "ram" to emulate SCM with memory, scm_list is ignored
-#  # Immutable after reformat.
-#
-#  scm_class: dcpm
-#
-#  # When scm_class is set to dcpm, scm_list is the list of device paths for
-#  # PMem namespaces (currently only one per engine supported).
-#  scm_list: [/dev/pmem0]
-#
-#  # Backend block device type. Force a SPDK driver to be used by this engine
-#  # instance.
-#  # Options are:
-#  # - "nvme" for NVMe SSDs (preferred option), bdev_size ignored
-#  # - "file" to emulate a NVMe SSD with a regular file
-#  # - "kdev" to use a kernel block device, bdev_size ignored
-#  # Immutable after reformat.
-#
-#  # When bdev_class is set to file, Linux AIO will be used to emulate NVMe.
-#  # The size of file that will be created is specified by bdev_size in GB units.
-#  # The location of the files that will be created is specified in bdev_list.
-#  bdev_class: file
-#  bdev_list: [/tmp/daos-bdev1,/tmp/daos-bdev2]
-#  bdev_size: 16
-#
-#  # When bdev_class is set to kdev, bdev_list is the list of unique kernel
-#  # block devices that should be different across different engine instance.
-#  bdev_class: kdev
-#  bdev_list: [/dev/sdc,/dev/sdd]
-#
-#  # If Volume Management Devices (VMD) are to be used, then the disable_vmd
-#  # flag needs to be set to false (default). The bdev_class will remain the
-#  # default "nvme" type, and bdev_list will include the VMD addresses.
-#  bdev_class: nvme
-#  bdev_list: ["0000:82:00.0","0000:5d:05.5"]
+#  storage:
+#  -
+#    # Define a pre-configured mountpoint for storage class memory to be used
+#    # by this engine.
+#    # Path should be unique to engine instance (can use different subdirs).
+#
+#    scm_mount: /mnt/daos/2
+#
+#    # Backend SCM device type. Either use PMem (Intel(R) Optane(TM) persistent
+#    # memory) modules configured in interleaved mode or emulate SCM with a
+#    # tmpfs running in RAM.
+#    # Options are:
+#    # - "dcpm" for real SCM (preferred option), scm_size is ignored
+#    # - "ram" to emulate SCM with memory, scm_list is ignored
+#    # Immutable after reformat.
+#
+#    class: dcpm
+#
+#    # When class is set to dcpm, scm_list is the list of device paths for
+#    # PMem namespaces (currently only one per engine supported).
+#    scm_list: [/dev/pmem0]
```

Comment:
This might be slightly confusing given the second engine is being assigned the first pmem device, I think it's more intuitive to map first and second pmem devices/sockets to first and second engines as it makes the resource allocation/upi usage mitigation story more obvious.